tests/tlsconfig_test.go

```diff
@@ -0,0 +1,141 @@
+package tests
+
+import (
+	"context"
+	"crypto/tls"
+	"encoding/json"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	api "github.com/kubevirt/vm-console-proxy/api/v1alpha1"
+	ocpconfigv1 "github.com/openshift/api/config/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/kubevirt/vm-console-proxy/pkg/console"
+)
+
+var _ = Describe("TLS config", func() {
+	const (
+		configMapName = "vm-console-proxy"
+	)
+
+	It("should read config from ConfigMap", func() {
```

Comment:
This implies the default config map was loaded? Can you make it more specific so the the config to be used is predetermined explicitly?

Reply:
Probably not easily. The config is created when the `vm-console-proxy` is installed.
I could do it in a future PR.

Reply:
Can you create an issue to track this follow up?

Reply:
Created: https://github.com/kubevirt/vm-console-proxy/issues/5